buildSrc/src/main/kotlin/conventions/Companion.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package conventions
+
+import org.gradle.accessors.dm.LibrariesForLibs
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.getByType
+
+// Adds the `libs` value for use in conventions
+internal val Project.libs: LibrariesForLibs
+    get() = extensions.getByType()
+
+/** Extension function to determine if a project property has been set. */
+fun Project.buildingWith(name: String): Boolean {
+    return this.hasProperty(name) && this.property(name).toString().toBoolean()
```

Comment:
seems the following would be better in terms of robustness (property value could still be null if it exists) and consistency with other similar usages:
```
return findProperty(name)?.toString().toBoolean()
```

Reply:
Done